src/main/java/core/basesyntax/LifeHackCalculus.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class LifeHackCalculus {
+    private static final int LINE_SEPARATOR_VALUE = 10;
+    private static final int SMALL_B_VALUE = 98;
+    private static final int SMALL_S_VALUE = 115;
+    private static final int SKIP_IF_S = 6;
+    private static final int SKIP_IF_B = 3;
+
+    private int supply;
+    private int buy;
+    private int result;
+    private final StringBuilder stringBuilder = new StringBuilder();
+
+    public void getCalculations(String fromFileName) {
+        int supply = 0;
+        int buy = 0;
```

Comment:
do not create duplicate variables, write directly to this.buy or to a local buy and return